### DISCUSSION TITLE
headings to sections of ticket questions

### DISCUSSION THREAD

**Discussion by felixrindt** [pretix/pretix#1972] [Ideas]
If someone uses a lot of questions, the presale interface quickly becomes cluttered and hard to navigate.
Therefore it would be cool to add section headings to questions to make clear that a set of questions is related (e.g. address (Postcode, City and street)).

**Comment by raphaelm** [comment]
I see why it might be useful, but it adds a lot of overhead to the control interface, doesn't it? Not sure it's justified.
